ecs/service-cluster-alb.yaml

```diff
@@ -165,6 +165,19 @@ Resources:
       - SubDomainNameWithDot: !Ref SubDomainNameWithDot
         HostedZoneName: {'Fn::ImportValue': !Sub '${ParentZoneStack}-HostedZoneName'}
       Type: A
+  RecordSetIPv6: # We can not conditionally create this only if the cluster's ALB has IPv6 turned on. Route53 does not let us query a broken AAAA record either. It just shows up as a Route53 record.
```

Comment:
`It just shows up as a Route53 record.` What does that mean?

Reply:
you see it in Route53 as a record but you can not resolve it.